src/core/Akka.Cluster/ClusterMetricsCollector.cs

```diff
@@ -856,18 +856,16 @@ public static IMetricsCollector Get(ExtendedActorSystem system, ClusterSettings
         {
             var fqcn = settings.MetricsCollectorClass;
             if (fqcn == typeof (PerformanceCounterMetricsCollector).AssemblyQualifiedName) return new PerformanceCounterMetricsCollector(system);
-            
-            var metricsCollectorClass = Type.GetType(fqcn);
-            if (metricsCollectorClass == null)
-            {
-                throw new ConfigurationException(string.Format("Could not create custom metrics collector {0}", fqcn));
-            }
-
             try
             {
-                var metricsCollector = (IMetricsCollector) Activator.CreateInstance(metricsCollectorClass, system);
+                var metricsCollectorClass = Type.GetType(fqcn, true);
+                var metricsCollector = (IMetricsCollector)Activator.CreateInstance(metricsCollectorClass, system);
```

Comment:
We should check that the type is a IMetricsCollector or throw configuration exception